examples/dag-with-args-example/README.md

```diff
@@ -124,6 +124,7 @@ def run(job_input: IJobInput):
     data_job_dir = pathlib.Path(job_input.get_job_directory())
     data_file = data_job_dir / "data.json"
 
+    db_catalog = job_input.get_arguments().get("db_catalog")
```

Comment:
The default value for the catalog is "memory" so IMO we should remove it in order to help people with the idea that they could switch to another DB if they want to and not necessarily stick to Trino. Moreover, it is not passed as argument anymore to the JOBS_RUN_ORDER dict so it would have no effect.

Reply:
dropped this